### PR TITLE
WIP: ABT-1762: added custom mail client

### PR DESCRIPTION
**Pull request by simeonpp**

### CODE REVIEW

**Comment by kalinkrustev** [comment]
This pattern of coding is not for ports, but for business modules. Ports define how a single port operates and do not export {ports, modules, validations} structure.

**Comment by simeonpp** [comment]
Updated port logic.

File: index.js [hunk 1]
```diff
@@ -1,81 +1,43 @@
...
     };
 }
 
-util.inherits(Mail, Port);
+util.inherits(MailPort, Port);
 
-Mail.prototype.init = function init() {
+MailPort.prototype.init = function init() {
     Port.prototype.init.apply(this, arguments);
-    this.latency = this.counter && this.counter('average', 'lt', 'Latency');
```
Comment (kalinkrustev) [inline index.js]: Are you sure that this should be removed?

**Review by kalinkrustev — CHANGES_REQUESTED** [review]
Let's also upgrade the versions of nodemailer modules.

**Pull request merged**